sources/packages/web/src/types/contracts/OfferingContact.ts

```diff
@@ -36,6 +38,8 @@ export interface OfferingDTO {
   offeringWILType?: string;
   studyBreaks?: StudyBreak[];
   offeringDeclaration: boolean;
+  offeringStatus: OfferingStatus;
```

Comment:
Are we resuing the API DTO to add more properties to accommodate some Vue UI needs?

Reply:
Since we have the new model `OfferingFormModel` we can now remove these properties from the DTO, right?

Reply:
offeringChipStatus now removed.